api/controllers/v0.3/UsernameValidationAPI.js

```diff
@@ -58,13 +58,14 @@ const UsernameValidationAPI = (args) => {
    * @param {string} username
    */
   const usernameAvailable = async (username) => {
+    const isBarcode = false;
```

Comment:
Why is this needed?

Reply:
The `available` function from the `ilsClient` can check barcode or username, but the flag is set so that it sets up the params correctly.